src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -259,9 +420,10 @@ private void OnWarmupEnded()
         userUpdate.Reward = new CrpgUserReward
         {
             Experience = (int)(serverXpMultiplier * durationRewarded * (_constants.BaseExperienceGainPerSecond
-                + crpgPeer.RewardMultiplier * _constants.MultipliedExperienceGainPerSecond)),
+            + crpgPeer.RewardMultiplier * _constants.MultipliedExperienceGainPerSecond)),
             Gold = (int)(durationRewarded * (_constants.BaseGoldGainPerSecond
-                + crpgPeer.RewardMultiplier * _constants.MultipliedGoldGainPerSecond)),
+            + crpgPeer.RewardMultiplier * _constants.MultipliedGoldGainPerSecond)
+            + compensationAmount),

```

Comment:
Please keep the indentation the way it was

Reply:
oops, was not intended
Done